release_ios.sh

```diff
@@ -20,13 +20,13 @@ fi
 make ios
 
 BUILD_DIR=build
-REPO_PATH="${BUILD_DIR}/matrix-wysiwyg"
+REPO_PATH="${BUILD_DIR}/matrix-wysiwyg-composer-swift"
```

Comment:
shouldn't this be the source repo?

Reply:
Is the name of the SPM repo that we clone in to.